app/javascript/views/orgs/admin_edit.js

```diff
@@ -7,21 +7,44 @@ import { eachLinks } from '../../utils/links';
 $(() => {
   const toggleFeedback = () => {
     const editor = Tinymce.findEditorById('org_feedback_email_msg');
+    const saveBtn = $('#save_org_submit');
     if (isObject(editor)) {
       if ($('#org_feedback_enabled_true').is(':checked')) {
         editor.setMode('code');
+        if (editor.getContent().trim() === '') {
```

Comment:
Did you try just adding the `"aria-required": true` to the field? We moved away from using JS to enforce form validations and instead rely on validations defined in the model (you can require the email body there with an `validates :feedback_email_msg, presence: true, if: { feedback_enabled: true }` clause and then displaying the appropriate success/failure to the user (see this section for an example: https://github.com/DMPRoadmap/roadmap/blob/4d9665ef8c76736ac750fe6ed0763da26be2a0f3/app/controllers/orgs_controller.rb#L57)

Reply:
@briri The functionality you mention was already correctly coded and working. (The bug was noted in July 2018, it appears to be fixed since then). To make the sample message copy-and-paste friendly I have moved the message that was in in the tooltip to be beside the input textarea.
![selection_122](https://user-images.githubusercontent.com/8876215/52784746-07ebff80-304e-11e9-8944-ed6c5ba98a71.png)